MainTel/MainTel-Troubleshooting/MTM2-Troubleshooting/MTM2_recovery.rst

```diff
@@ -0,0 +1,306 @@
+
+    #. Establish **local control** with the EUI by hitting connect, then local (local may be greyed out after hitting connect, this is ok).
+
+        .. figure:: _static/MTM2_recovery_GUI_open_connect.png
+            :name: MTM2 GUI open and connect
+            :width: 600
+            :align: center
+
+    #. Pull up the **Overview widget** by double-clicking on Overview in the list at the bottom of the EUI.
+
+        .. figure:: _static/MTM2_recovery_GUI_overview.png
+            :name: MTM2 GUI Overview
+            :width: 600
+            :align: center
+
+        #. Observe the **Enabled Faults Mask**. It should **not be 0**. If it is, repeat :ref:`reset GIS signal <MTM2-recovery-Reset-the-M2-interlock-signal>`.
+
+            .. note::
+                
+                It is ok if the ``isInterlockEngaged`` indicator is red.  
+
+                .. the original said isInterlockEnabled
+
+        #. You may also look at **Alarms/Warnings** widget to see active alarms (red) or warnings (yellow). If active, reset them with **Reset All Items**. Note: *Make sure you have removed the fault condition*.
+
+            .. figure:: _static/MTM2_recovery_GUI_alarms_warnings.png
+                :name: GUI Alarms and warnings widget 
+                :width: 800
+                :align: center
+
+            #. If Reset All Items does not work, you maybe have to `power cycle M2 cabinet`_ *Only do this if there are no other options!*
+
+    #. Move to Diagnostic mode by hitting the :guilabel:`Diagnostic` button, be patient this may take some time.
+
+    #. Move to Enabled by hitting the :guilabel:`Enabled` button, be patient this may take up to 2 minutes.
+
+        #. If this step fails, you may have to repeat :ref:`reset GIS signal <MTM2-recovery-Reset-the-M2-interlock-signal>`.
+
+    #. Move to Closed-Loop control by pressing the :guilabel:`Enter closed-loop control` button.
+
+#. **Return to standby in EUI to close GUI**:
+
+    #. Return to open-loop control by pressing the :guilabel:`Enter open-loop control` button.
+
+    #. Press the :guilabel:`Diagnostic` button to enter Diagnostic mode, this usually takes ~30s.
+
+    #. Press the :guilabel:`Standby`` button to enter Standby, this usually takes ~30s.
+    
+    #. Disconnect EUI by pressing disconnect on the top tool bar, this usually takes ~30s.
+    
+    #. Exit EUI by pressing exit on the top tool bar.
+
+#. Move CSC to ``ENABLED``:
+
+    #. Once the CSC is in ``Disabled`` wait 2 minutes.
+
+    #. Transition the CSC to ``Enabled`` using enable, this may take up to 2 minutes.
+
+    #. If it fails, move the CSC all the way to ``standby`` and try to transition to ``enabled`` again with the same wait times.
+
+#. Check that M2 in under Close Loop control **4** in Chronograf `M2 status`_.
+
+    #. If needed, set Closed Loop control by following the ``run_command`` configuration found in `LOVE MT Useful lower-level command scripts + configurations`_.
```

Comment:
There's now a SAL Script that without configuration enters *Closed loop* ``maintel/m2/enable_closed_loop.py`` It's also in https://confluence.lsstcorp.org/pages/viewpage.action?pageId=239409017